src/org/zaproxy/zap/extension/httpsinfo/resources/help/contents/httpsinfo.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+HTTPS Info Extension
```

Comment:
"Add-on" instead of "Extension"? (same comment for other entries in this file)
We should normalise how we refer it/them :)